app/reducers/rooms.ts

```
@@ -44,6 +56,37 @@ export default function rooms(state = initialState, action: IRoomsAction): IRoom
 				refreshing: true
 			};
+		case ROOMS.STORE_LAST_VISITED:
+			if (!action.lastVisitedRoomId || !action.server) {
+				return state;
+			}
+
+			const newRoom: IRecentRoomsStore = {
+				rid: action.lastVisitedRoomId,
+				name: action.lastVisitedRoomName,
+				server: action.server
+			};
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)